java/runtime-native/src/main/java/org/ray/spi/impl/DefaultLocalSchedulerClient.java

```diff
@@ -64,6 +65,22 @@ private static native long _init(String localSchedulerSocket, byte[] workerId,
 
   @Override
   public void submitTask(TaskSpec task) {
+    // We don't support resources management in non raylet mode.
+    if (!useRaylet) {
+      task.resources.clear();
+      task.resources.put("CPU", 0.0);
+    } else {
+      final String cpuLiteral = "CPU";
```

Comment:
I think `"CPU"` should be as a constant. 
I prefer creating a `ResourceUtil` class, and put the constants and the helper functions in this class.

Reply:
It's a wonderful suggestion.